.github/workflows/unittests.yml

```diff
@@ -56,6 +56,10 @@ jobs:
           sudo apt install -y attr
           sudo mkdir -p /home/travis/.m2/
 
+      - name: Install Fdb C Lib
+        run: |
+          wget https://github.com/apple/foundationdb/releases/download/7.1.7/foundationdb-clients_7.1.7-1_amd64.deb
```

Comment:
Use latest stable version from Github (6.3.24). 
Also please try installing the server here to make it possible to run `TestFdbClient`